lib/tests/filesystem.sh

```diff
@@ -38,11 +38,15 @@ mkfifo fifo
 checkPathType() {
     local path=$1
     local expectedPathType=$2
-    local actualPathType=$(nix-instantiate --eval --strict --json 2>&1 \
+    local result
+    stderr=$(mktemp)
```

Comment:
This file should be cleaned up, because the tests should run cleanly outside the sandbox. Maybe you could put it in the `$work` directory?